Research/Research/RSDTaskViewModel.swift

```diff
@@ -0,0 +1,947 @@
+//
+//  RSDTaskViewModel.swift
+//  Research
+//
+//  Copyright © 2017-2018 Sage Bionetworks. All rights reserved.
```

Comment:
...I'm guessing from this that you renamed and edited an existing file, and now Github thinks it's a whole new file. Thanks Github. :/